.github/workflows/release.yml

```diff
@@ -0,0 +1,88 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        # linux
+        - os: ubuntu-latest
```

Comment:
We actually do not have any platform specific code included. As such we do not need separate build artifacts for the release. It will be enough to provide the tar'ed source package and the wheel. That means it will be enough to run this job for Linux and for the latest supported Python version.

Reply:
Thank you, I had that question.